tests/common/fixtures/advanced_reboot.py

```diff
@@ -40,6 +40,14 @@ def __init__(self, request, duthost, ptfhost, localhost, tbinfo, creds, **kwargs
         )
 
         if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
```

Comment:
Please add a detailed comments here to state why this change is needed.

Reply:
Updated now, thanks!